filters/apiusagemonitoring/pathinfo.go

```diff
@@ -23,7 +22,7 @@ type pathInfo struct {
 	metricPrefixedPerClient sync.Map
 }
 
-func newPathInfo(applicationId, tag, apiId, pathTemplate, pathLabel string, clientTracking *clientTrackingInfo) *pathInfo {
+func newPathInfo(applicationId, tag, apiId, pathLabel string, clientTracking *clientTrackingInfo) *pathInfo {
```

Comment:
would it be worth renaming `pathLabel` to `pathTemplate`?

Reply:
Overlooked :)